crypto/keyring/keyring.go

```diff
@@ -100,13 +100,6 @@ type Keyring interface {
 	Migrator
 }
 
-// UnsafeKeyring exposes unsafe operations such as unsafe unarmored export in
-// addition to those that are made available by the Keyring interface.
-type UnsafeKeyring interface {
```

Comment:
The keyring has a lot of `Unsafe*` stuff. They are only used for exporting the private key. This is why I moved all Unsafe* types into `client/keys/export.go`.

Reply:
If you think callers won't need these, then this is fine 👍